### ISSUE TITLE
Hemidiaphragm elevation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10206]
**Preferred term label:**
Hemidiaphragm elevation

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C3554250 for "Elevated right hemidiaphragm" bit I don't see anything for left or either.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shrinking lung syndrome (which can be on either left or right)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000008
- label: Hemidiaphragm elevation
- parents: HP:0040045
- pmid: PMID:32644681
- def: Elevated Hemidiaphragm is a condition where one portion of the diaphragm is higher than the other. Elevated hemidiaphragm occurs when one side of the diaphragm becomes weak from muscular disease or loss of innervation due to phrenic nerve injury.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10206
- synonyms: Elevated hemidiaphragm(NONE)
